x-pack/plugins/lens/public/xy_visualization/xy_config_panel.tsx

```diff
@@ -68,3 +80,72 @@ export function LayerContextMenu(props: VisualizationLayerWidgetProps<State>) {
     </EuiFormRow>
   );
 }
+
+export function XyToolbar(props: VisualizationToolbarProps<State>) {
+  const [open, setOpen] = useState(false);
+  const hasNonBarSeries = props.state?.layers.some(
+    (layer) => layer.seriesType === 'line' || layer.seriesType === 'area'
+  );
+  return (
+    <EuiFlexGroup justifyContent="flexEnd">
+      <EuiFlexItem grow={false}>
+        <EuiPopover
+          panelClassName="lnsXyToolbar__popover"
+          button={
+            <EuiButtonEmpty
+              iconType="arrowDown"
+              iconSide="right"
+              onClick={() => {
+                setOpen(!open);
+              }}
+            >
+              <EuiIcon type="gear" />
+            </EuiButtonEmpty>
+          }
+          isOpen={open}
+          closePopover={() => {
+            setOpen(false);
+          }}
+          anchorPosition="downRight"
+        >
+          <EuiFormRow
+            display="columnCompressed"
+            label={i18n.translate('xpack.lens.xyChart.fittingLabel', {
+              defaultMessage: 'Missing values',
+            })}
```

Comment:
I don't think that "Missing Values: None", which is the default state for us, is easy to understand.

<img width="383" alt="Screenshot 2020-06-30 18 21 40" src="https://user-images.githubusercontent.com/666475/86182823-aa4f3a80-bafe-11ea-9384-47834a3d85cd.png">

I would prefer one of these two solutions:

1. Change the default to "Missing values: Zero", so users understand the behavior
2. Change the text to either "Fit missing values: None" or "Missing values: Hidden"